### PR TITLE
Update the RFC templates and README to reflect RFC Process Updates RFC

### PR DESCRIPTION
**Pull request by kategengler**
[#300](https://github.com/emberjs/rfcs/blob/master/text/0300-rfc-process-update.md)

### CODE REVIEW

File: 0000-template.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 - Start Date: (fill me in with today's date, YYYY-MM-DD)
-- RFC PR: (leave this empty)
-- Ember Issue: (leave this empty)
+- Relevant Team(s): (fill this in with the team(s) to which this RFC applies)
```
Comment (rwjblue) [inline 0000-template.md]: Cross-link to list in the README?

File: README.md [hunk 2]
```diff
@@ -15,9 +15,10 @@ consistent and controlled path for new features to enter the framework.
 ## When you need to follow this process
 
 You need to follow this process if you intend to make "substantial"
-changes to Ember, Ember Data or its documentation. What constitutes a
-"substantial" change is evolving based on community norms, but may
-include the following.
+changes to Ember, Ember Data, Ember CLI, their documentation, or any other 
+projects under the purview of the [Ember core teams](https://emberjs.com/team/)
```
Comment (rwjblue) [inline README.md]:
```suggestion
 projects under the purview of the [Ember core teams](https://emberjs.com/team/).
```

File: README.md [hunk 3]
```diff
@@ -15,9 +15,10 @@ consistent and controlled path for new features to enter the framework.
 ## When you need to follow this process
 
 You need to follow this process if you intend to make "substantial"
-changes to Ember, Ember Data or its documentation. What constitutes a
-"substantial" change is evolving based on community norms, but may
-include the following.
+changes to Ember, Ember Data, Ember CLI, their documentation, or any other 
+projects under the purview of the [Ember core teams](https://emberjs.com/team/) 
+What constitutes a "substantial" change is evolving based on community norms, 
+but may include the following.
```
Comment (rwjblue) [inline README.md]:
```suggestion
 but may include the following:
```

File: README.md [hunk 4]
```diff
@@ -45,51 +46,92 @@ It's often helpful to get feedback on your concept before diving into the
 level of API design detail required for an RFC. **You may open an
 issue on this repo to start a high-level discussion**, with the goal of
 eventually formulating an RFC pull request with the specific implementation
-design.
+design. We also highly recommend sharing drafts of RFCs in `dev-rfc` on
```
Comment (rwjblue) [inline README.md]:
```suggestion
 design. We also highly recommend sharing drafts of RFCs in `#dev-rfc` on 
```

File: README.md [hunk 5]
```diff
@@ -113,15 +155,53 @@ If you are interested in working on the implementation for an 'active'
...
+### Helpful checklists for Champions
+
+#### Becoming champion of an RFC
+- [ ] Assign the RFC to yourself
+
+#### Moving to FCP to Merge
+- [ ] Achieve consensus to move to "FCP to Merge" from relevant core teams
+- [ ] Comment in the RFC to address any outstanding issues and to proclaim the 
+start of the FCP period
+- [ ] Tweet from `emberjs` about the FCP
```
Comment (rwjblue) [inline README.md]:
```suggestion
 - [ ] Tweet from `@emberjs` about the FCP 
```

File: deprecation-template.md [hunk 6]
```diff
@@ -1,6 +1,7 @@
 - Start Date: (fill me in with today's date, YYYY-MM-DD)
-- RFC PR: (leave this empty)
-- Ember Issue: (leave this empty)
+- Relevant Team(s): (fill this in with the team(s) to which this RFC applies)
```
Comment (rwjblue) [inline deprecation-template.md]: Maybe cross link to the actual list of teams?

File: 0000-template.md [hunk 7]
```diff
@@ -1,6 +1,7 @@
...
+- RFC PR: (after opening the RFC PR, update this with a link to it)
```
Comment (rwjblue) [inline 0000-template.md]: Might want to mention renaming the file also?

File: 0000-template.md (hunk 1 above)
Comment (locks) [inline 0000-template.md]:
```suggestion
- Relevant Team(s): (fill this in with the team(s)–https://emberjs.com/team/–to which this RFC applies)
```

File: 0000-template.md (hunk 7 above)
Comment (locks) [inline 0000-template.md]:
```suggestion
- RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
```

File: README.md [hunk 8]
```diff
@@ -45,51 +46,92 @@ It's often helpful to get feedback on your concept before diving into the
...
 In short, to get a major feature added to Ember, one must first get the
 RFC merged into the RFC repo as a markdown file. At that point the RFC
 is 'active' and may be implemented with the goal of eventual inclusion
 into Ember.
 
 * Fork the RFC repo http://github.com/emberjs/rfcs
-* Copy `0000-template.md` to `text/0000-my-feature.md` (where
+* Copy the appropriate template. For most RFCs, this is `0000-template.md`, 
+for deprecation RFCs it is `deprecation-template.md`.
+Copy the template file to `text/0000-my-feature.md` (where
```
Comment (locks) [inline README.md]:
```suggestion
Copy the template file to `text/0000-my-feature.md`, where
```

File: README.md [hunk 9]
```diff
@@ -45,51 +46,92 @@ It's often helpful to get feedback on your concept before diving into the
...
 'my-feature' is descriptive. don't assign an RFC number yet).
```
Comment (locks) [inline README.md]:
```suggestion
'my-feature' is descriptive. Don't assign an RFC number yet.
```

File: README.md [hunk 10]
```diff
@@ -45,51 +46,92 @@ It's often helpful to get feedback on your concept before diving into the
...
 * Fill in the RFC. Put care into the details: **RFCs that do not
 present convincing motivation, demonstrate understanding of the
 impact of the design, or are disingenuous about the drawbacks or
 alternatives tend to be poorly-received**.
+* Fill in the relevant core teams. Use the below table to map from projects to
```
Comment (locks) [inline README.md]:
```suggestion
* Fill in the relevant core teams. Use the table below to map from projects to 
```

File: README.md [hunk 11]
```diff
@@ -113,15 +155,53 @@ If you are interested in working on the implementation for an 'active'
...
-We try to make sure that any RFC that we accept is accepted at the
-Friday team meeting, and reported in [core team notes]. Every
-accepted feature should have a core team champion, who will represent
-the feature and its progress.
+## For Core Team Members
+
+### Reviewing RFC's
+
+Each core team is responsible for reviewing open RFCs. If an RFC 
+requires work from their team, ensuring that the RFC reflects that.
```
Comment (locks) [inline README.md]:
```suggestion
requires work from their team, the RFC reflects that. 
```

File: deprecation-template.md (hunk 6 above)
Comment (locks) [inline deprecation-template.md]:
```suggestion
- Relevant Team(s): (fill this in with the team(s)–https://emberjs.com/team/–to which this RFC applies)
```

File: deprecation-template.md [hunk 12]
```diff
@@ -1,6 +1,7 @@
...
+- RFC PR: (after opening the RFC PR, update this with a link to it)
```
Comment (locks) [inline deprecation-template.md]:
```suggestion
- RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
```

File: 0000-template.md (hunk 1 above)
Comment (kategengler) [reply 0000-template.md]: Should this link to emberjs.com/team or to the README table as rwjblue suggested?

File: README.md (hunk 11 above)
Comment (kategengler) [reply README.md]: Removing those words changes the meaning here, but I'll update to make my intention more clear (it was a sloppy sentence to begin with 😬).

Something like:
"Each team must ensure that any RFC that requires work from their team appropriately states that fact"

File: README.md [hunk 13]
```diff
@@ -113,15 +155,64 @@ If you are interested in working on the implementation for an 'active'
...
+- [ ] Comment in the RFC to explain the decision
 
-Each week the [core team] will attempt to review some set of open RFC
-pull requests.
+#### Closing an RFC
+- [ ] Comment about the end of the FCP period with no new info
+- [ ] Close the PR
 
-We try to make sure that any RFC that we accept is accepted at the
-Friday team meeting, and reported in [core team notes]. Every
```
Comment (acorncom) [inline README.md]: at the **appropriate** team meeting?

Comment (kategengler) [reply README.md]: 😆 Just have the 👶 review instead

**Pull request merged**